src/themes/eu/variables/_text-area.scss

```diff
@@ -2,6 +2,7 @@
 @use '../index' as *;
 
 $textarea: (
+  font: var(--f-ui-m),
```

Comment:
i thought you had inverted the values between EC and EU, instead you set --f-ui in both and with the same value, all of this was not needed at all, but since you did it, please ensure that you are not altering anything that is not supposed to be updated.